front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/index.tsx

```diff
@@ -47,12 +48,26 @@ const ContentBuilderToolbox = ({
           ref &&
           connectors.create(
             ref,
-            <Element canvas is={TwoColumn} id="twoColumn" />
+            <Element canvas is={TwoColumn} columnLayout="1-1" id="twoColumn" />
           )
         }
       >
         <ToolboxItem icon="column2" label={formatMessage(messages.twoColumn)} />
       </DraggableElement>
+      <DraggableElement
+        ref={(ref) =>
+          ref &&
+          connectors.create(
+            ref,
+            <Element canvas is={ThreeColumn} id="threeColumn" />
+          )
+        }
+      >
+        <ToolboxItem
+          icon="column2"
```

Comment:
This is because we don't have the icon, correct?

Reply:
Correct - I won't merge this until next week when I can swap it out for a new column3 icon :+1: